DevOps/Nginx/Nginx_User_Permission_Restriction.md

```
@@ -0,0 +1,161 @@
+    add_header Content-Disposition inline;
+}
+
+```
+
+}
+
+In above code i want to restrice location with id and password for that
+
+eg➖
+
+# Location block that defines the behavior for requests to this server.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)